### DISCUSSION TITLE
Is "dumb" trainer a use case?

### DISCUSSION THREAD

**Discussion by DavideRossi** [dvmarinoff/Auuki#137] [Q&A] [answered]
Flux seems clearly targeted at smart trainers.
But I do not have a smart trainer and I'm trying to understand if Flux can be useful with my "dumb" trainer (with a good-old manual resistance selector) if I throw in the mix a heart band and a speed/cadence sensor.
Suggestions?

**Comment by dvmarinoff** [comment, marked as answer]
I guess usage is limited this way. You can still upload .fit rides and courses and ride them with just speed. Calculating power from speed and cadence thought won't be supported.
